src/locations/index.ts

```diff
@@ -54,39 +54,64 @@ export async function getLocations(options: LocationCmdOptions) {
 }
 
 export function formatLocations(locations: Array<LocationMetadata>) {
-  const formatted: Array<string> = [];
+  const publicLocations: Array<string> = [];
+  const privateLocations: Array<string> = [];
   for (const location of locations) {
     let name = location.label;
     if (location.isServiceManaged) {
       [, name] = name.includes('-') ? name.split('-') : [, name];
       name = name.toLowerCase().trim().split(' ').join('_');
+      publicLocations.push(name);
     } else {
-      name = `${name}${PRIVATE_KEYWORD}`;
+      privateLocations.push(name);
     }
-    formatted.push(name);
   }
-  return formatted;
+  return {
+    publicLocations,
+    privateLocations,
+    allLocations: [
+      ...publicLocations,
+      ...privateLocations.map(loc => `${loc} ${PRIVATE_KEYWORD}`),
+    ],
+  };
 }
 
 export function groupLocations(locations: Array<string>) {
```

Comment:
this function only existed because we were merging both locations together, now that you have splitted the locations in to distinct categories, we dont need to split and join again with Private Keyword.

Reply:
Had a chat offline, this was done to make `locations` command look nicer with different types of locations, we still will use the keyword on init.